cmd/podman/ps.go

```diff
@@ -64,7 +64,7 @@ type psJSONParams struct {
 	Status           string                        `json:"status"`
 	PID              int                           `json:"PID"`
 	Ports            []ocicni.PortMapping          `json:"ports"`
-	Size             *batchcontainer.ContainerSize `json:"size,omitempty"`
```

Comment:
Why drop the omitempty? We don't want to put unnecessary nils in our JSON output

Reply:
If I ask for size, and I don't see size, then that is a bug.

Reply:
This puts it in the JSON unconditionally, even when you don't ask for it. I don't see many people using the JSON output for human-readable output, either - what does it matter if it's there or not if it's just a nil and the other side is a JSON decoder?

Reply:
Not worth arguing over.  The main issue I was fixing is the size failure.